portfolio/src/main/webapp/index.html

```diff
@@ -49,25 +49,25 @@ <h1>Michaela Murray's Portfolio</h1>
         <div class="mdc-touch-target-wrapper">
           <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank">
             <button class="mdc-button mdc-button--outlined">  
-	       <span class="mdc-button__ripple">Resume</span> 
-	    </button>
+	            <span class="mdc-button__ripple">Resume</span> 
+	        </button>
           </form>
         </div>
 	<div id="icons">
             <ul> 
             <li> 
-                <a href="https://github.com/mmurray22" target="_blank">
-                    <img id="icon" src="/images/logo-github.svg">
+                <a href="https://github.com/mmurray22" target="_blank" alt="Github icon">
+                    <img class="icon" src="/images/logo-github.svg">
                 </a>
             </li> 
             <li>
-                <a href="https://www.linkedin.com/in/murray22/" target="_blank">
-                    <img id="icon" src="/images/logo-linkedin.svg">
+                <a href="https://www.linkedin.com/in/murray22/" target="_blank" alt="LinkedIn Icon">
+                    <img class="icon" src="/images/logo-linkedin.svg">
                 </a>
             </li> 
             <li> 
                 <a href="mailto:murray22@stanford.edu">
-                    <img id="icon" src="/images/email-24px.svg">
+                    <img class="icon" src="/images/email-24px.svg" target="_blank" alt="Email Icon">
```

Comment:
I don't think `target` is a valid attribute for `img`. Did you mean for this to go on the anchor?

Reply:
yes! updated!